GameDataParser/Parsers/ConstantsParser.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Xml;
+using GameDataParser.Files;
+using Maple2.File.IO.Crypto.Common;
+using Maple2Storage.Types.Metadata;
+
+namespace GameDataParser.Parsers
+{
+    public class ConstantsParser : Exporter<List<ConstantsMetadata>>
+    {
+        public ConstantsParser(MetadataResources resources) : base(resources, "constants") { }
+
+        protected override List<ConstantsMetadata> Parse()
+        {
+            List<ConstantsMetadata> constants = new List<ConstantsMetadata>();
+            foreach (PackFileEntry entry in Resources.XmlReader.Files)
+            {
+                if (!entry.Name.StartsWith("table/constants"))
+                {
+                    continue;
+                }
+
+                XmlDocument document = Resources.XmlReader.GetXmlDocument(entry);
+                foreach (XmlNode node in document.DocumentElement.ChildNodes)
+                {
+                    string locale = string.IsNullOrEmpty(node.Attributes["locale"]?.Value) ? "" : node.Attributes["locale"].Value;
+                    if (locale != "NA" && locale != "")
```

Comment:
Not sure if I'm over-thinking this but if you do `if (locale != "NA"` then `locale` must be `NA` to pass this check, and if so, then you don't need to also check if `locale != ""`, right?

Reply:
It needs both. There's several shared (between regions) values with empty locale.